src/shared/battery-util.c

```diff
@@ -254,19 +254,28 @@ int battery_is_discharging_and_low(void) {
 
                 level = battery_read_capacity_percentage(dev);
                 if (level < 0) {
+                        log_device_debug_errno(dev, level, "Battery capacity is unreadable: %m");
```

Comment:
this is pretty unnecessary, battery_read_capacity_percentage() debug logs on its own about this already, in more detailed way?